aldryn_config.py

```diff
@@ -191,6 +193,17 @@ def logging_settings(self, settings, env):
             }
         }
 
+    def sentry_settings(self, settings, env):
+        sentry_dsn = env('SENTRY_DSN')
+
+        if sentry_dsn:
+            settings['INSTALLED_APPS'].append('raven.contrib.django')
+            settings['RAVEN_CONFIG'] = {'dsn': sentry_dsn}
+            settings['LOGGING']['handlers']['sentry'] = {
```

Comment:
user wouldn't be able to, this gets called as soon as logging is setup with self.logging_settings, once all of this runs then user is free to override anything on the settings.py in project